config/config.msft.yaml

```diff
@@ -30,6 +29,12 @@ defaults:
     etcd:
       kvName: arohcp-etcd-{{ .ctx.regionShort }}
       kvSoftDelete: true
+    istio:
+      url: "https://github.com/istio/istio/releases/download"
+      istioctlVersion: "1.23.1"
+      tag: "prod-stable"
+      targetVersion: ""
+      version: "asm-1-22"
```

Comment:
Doesn't this need to be an array?

Reply:
More to the point - we had talked about having an array here of the versions of the istio control plane installed on the cluster. So the upgrade steps would look like

```
# step 0, current state
istio:
  versions: ["asm-1-22"]
  targetVersion: "asm-1-22"

# step 1, begin upgrade by installing new control plane and using it
istio:
  versions: ["asm-1-22", "asm-1-23"]
  targetVersion: "asm-1-23"

# step 2, clean up previous
istio:
  versions: ["asm-1-23"]
  targetVersion: "asm-1-23"
```